commons/push/integration/garmin/garmin_push_stress_level_sample.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.push.integration.garmin",
+  "name": "GarminStressLevelSample",
+  "type": "record",
+  "doc": "Stress levels are provided as 3-minute averages of the real-time stress scores generated on the device with values ranging from 1 to 100. A value of -1 means there was not enough data to detect stress, and -2 means there was too much motion (e.g. the user was walking or running). Scores between 1 and 25 are considered “rest” (i.e. not stressful), 26-50 as “low” stress, 51-75 “medium” stress, and 76- 100 as “high” stress. These numbers are derived based on a combination of many device sensors and will automatically adjust to the wearer of the device and gain accuracy over time as the stress algorithms learn the user’s natural biometric norms. Usually assocaited with Stress Details Summary using the summaryId. Stress values from the Health API are exactly the stress values shown on Garmin Connect.",
+  "fields": [
+    {"name": "summaryId", "type": "string", "doc": "Unique identifier for the summary that this sample is associated with."},
+    {"name": "time", "type": "double", "doc": "Start time of the sample in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time this sample was recieved by the push service in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "stressLevel", "type": ["null", "double"], "doc": "Stress level value recorded for the time..", "default": null}
```

Comment:
Is this value derived in any documented way? Could be a description or a URL?

Reply:
The only documentation is as mentioned in the record-level doc - `These numbers are derived based on a combination of many device sensors and will automatically adjust to the wearer of the device and gain accuracy over time as the stress algorithms learn the user’s natural biometric norms.`